src/common/snippets/include/snippets/generator.hpp

```diff
@@ -117,7 +117,7 @@ class Generator {
      * @param m model in canonical for for table-based code generation
      * @return pointer to generated code
      */
-    code generate(std::shared_ptr<ov::Model>& m, const void* compile_params = nullptr) const;
+    code generate(std::shared_ptr<ov::Model>& m, const void* compile_params = nullptr, const bool need_check_rt_info = false) const;
```

Comment:
Yes, I see your point totally.
We can just pass the whole snippets config, this would look like a more extendable solution. Probably there is no need for a special generation config at this point.
The original reason for this remark is that we both check rt_info and snippets config. Do we need both? Please see the relevant comment in generator.cpp also.

Reply:
As discussed now we move whole config. Thanks!